common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -440,7 +453,7 @@ class @Problem
         @el.addClass 'showed'
         @el.find('.show').attr('disabled', 'disabled')
         @updateProgress response
-        window.SR.readElts(answer_text)
+        window.SR.readText(gettext('The answer to the problem has been shown. Navigate through the problem to read the answer text.'))
```

Comment:
@catong I like the second one!  How about this: "Answers to this problem are now shown inline. Navigate through the problem to review."

Added "inline" removed "answers" from the end.